libs/java/core/scripts/close-staging-repo.sh

```diff
@@ -5,6 +5,6 @@ read -p 'Repository Id: cabcgovtno-' varStagingRepoId
 
 if [ ! -z "$varStagingRepoId" ]
 then
-  cd dal-db
+  export MAVEN_OPTS="--add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.lang.reflect=ALL-UNNAMED --add-opens=java.base/java.text=ALL-UNNAMED --add-opens=java.desktop/java.awt.font=ALL-UNNAMED"
```

Comment:
This resolves a bug with Maven and must be executed at least once during the command session.  I've simply added it to each script to ensure it works now matter what is called.